components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,28 @@
+import classNames from "classnames";
+import styles from "./TextInput.module.scss";
+
+function TextInput({
+  id, label, hideLabel, value, onChange, placeholder
+} : {
+  id?: string,
+  label: string,
+  hideLabel?: boolean,
+  value?: string,
+  onChange?: () => void,
+  placeholder?: string
```

Comment:
we'll have to make sure this component can accept {...register('inputName', {...config})} from `react-hook-form`

Reply:
likely can add `import { UseFormRegister } from 'react-hook-form'` and use that as the register trype

Reply:
this tutorial seems decent: https://www.thisdot.co/blog/how-to-create-reusable-form-components-with-react-hook-forms-and-typescript

Reply:
pushed up a [lil commit](https://github.com/populist-vote/web/pull/39/commits/20178b226416243d8c3a3ae3788bff4d2645ee0d) with this...feel free to change if you see fit